### ISSUE TITLE
Fix for Captcha issue

### ISSUE THREAD

**Issue by emhl** [jmir1/overleaf-sync#18]
@moritzgloeckl created a workaround to still sync files, even with captcha login

https://github.com/moritzgloeckl/overleaf-sync

**Comment by jmir1** [comment]
yeah, i wish i could fix this, but this is supposed to be a completely automatic github action, and they dont really support that kind of user input if i'm not mistaken.

not sure how to go around doing this

**Comment by DavidBerdik** [comment]
Why not do this?

1. Write a small application that asks users to log in to their Overleaf account and captures a session token.
2. Instruct users to make the session token a repo secret.
3. Have this GitHub action use the session token to authenticate to the Overleaf account.

I know it's not ideal to have to go through this extra step, but it would be more reliable than the sign-in method used here.
